### DISCUSSION TITLE
Is it possible to cut off/only show half the graph?

### DISCUSSION THREAD

**Discussion by Eneswar** [DylanHojnoski/obsidian-graphs#52] [Q&A]
![image](https://github.com/user-attachments/assets/e31cf310-ac82-4712-95c2-351360be9859)

Like the image shows, is it possible? I would like to just see the y-axis on the left.

**Comment by DylanHojnoski** [comment]
You can change the x min bounds to 0 and then it won't show the left side of the graph.

**Reply by Eneswar** [reply]
I didnt even think about trying that, thanks :D
